test/fixtures/input/flink/flinkshell-commands

```diff
@@ -0,0 +1,11 @@
+use catalog default;
```

Comment:
Can integration tests test only one (or if necessary, two) statement at a time. From previous experience, once you start chaining multiple statements/commands in each test it gets harder to debug.

Reply:
yeah we can split it up a bit, I was thinking that it is faster this way, since we don't have to do the setup all over again.